spirv_msl.cpp

```diff
@@ -3839,6 +3842,12 @@ string CompilerMSL::builtin_to_glsl(BuiltIn builtin, StorageClass storage)
 		return "gl_VertexIndex";
 	case BuiltInInstanceIndex:
 		return "gl_InstanceIndex";
+	case BuiltInBaseVertex:
+		return "gl_BaseVertex";
+	case BuiltInBaseInstance:
+		return "gl_BaseInstance";
+	case BuiltInDrawIndex:
+		return "gl_DrawID";
```

Comment:
@cdavis5e For consistency...can you also throw the same "DrawIndex is not supported in MSL" exception here, too, please? There is no point outputting the `gl_DrawID` variable if it can't be used and will throw a Metal compilation exception later.